src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -40,7 +40,7 @@ export default Object.freeze({
   form526BDD: 'form526BenefitsDeliveryAtDischarge',
   showEduBenefits5495Wizard: 'showEduBenefits5495Wizard',
   showEduBenefits1995Wizard: 'showEduBenefits1995Wizard',
-  showEduBenefits1990NWizard: 'showEduBenefits1990NWizard',
+  showEduBenefits1990NWizard: 'show_edu_benefits_1990n_wizard',
```

Comment:
This will either need to be updated to be `1990N` or the feature toggle name in the API will need to be updated to be `1990n`.